src/abstractions/cipher.service.ts

```diff
@@ -12,6 +12,7 @@ import { FieldView } from '../models/view/fieldView';
 
 export abstract class CipherService {
     decryptedCipherCache: CipherView[];
+    topUsernames: string[];
```

Comment:
Please use a consistent naming convention rather than mixing `default` and `top`. `commonUsernames` seems the most descriptive to me, but as long as its consistent I don't mind. This change should be replicated across all PRs.